opencog/rule-engine/InferenceSCM.cc

```diff
@@ -79,16 +80,17 @@ void InferenceSCM::init(void)
 #endif
 }
 
-Handle InferenceSCM::do_forward_chaining(Handle h)
+Handle InferenceSCM::do_forward_chaining(Handle h, const string& conf_path)
```

Comment:
BTW, it's a just a matter of days before I get rid of JSON (say Monday if I'm lucky).